rector.php

```diff
@@ -32,5 +31,8 @@
     $rectorConfig->fileExtensions(['php', 'module', 'theme', 'install', 'profile', 'inc', 'engine']);
     $rectorConfig->importNames(true, false);
     $rectorConfig->importShortClasses(false);
-    $parameters->set('drupal_rector_notices_as_comments', true);
+
+    $rectorConfig->singleton(AddCommentService::class, function() {
+        return new AddCommentService(true);
+    });
 };
```

Comment:
Can we add this in the root level of 8.0 or 9.0 config list to prevent a re-copy?

Reply:
We could probably, but we need recopy for removed ->arguments method anyways. 

Also i do want to be able to disable it in update bot. But that might still work, needs testing though.

Reply:
if that's the way it is, so be it.

Configurations with `parameters = $rectorConfig->parameters();` will return an error I'm guessing?

I really wish there was a way to make this easier.

Reply:
Yeah it errors, doesnt exist anymore.

What if we include comment service somewhere else? Perhaps we can make it part of the rules? Not sure if rulesets might have access to the container... Im not sure what our other options are.

Reply:
Hmm, adding the service in the ruleset might actually work... Then i can still disable it in the rector.php by setting the concrete implementation and would mean we do not need to do this in the future? Let me try that